test/client/js/libs/handlebars/parse.js

```diff
@@ -312,6 +325,10 @@ describe("parse Handlebars helper", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should find emoji", function() {
+		expect(parse("Hello💬")).to.deep.equal('Hello<span class="emoji">💬</span>');
```

Comment:
Can you add more tests like emojis in links, channels, irc colored emojis?

Reply:
Yes, will do after work.

Reply:
Done.